README.md

```diff
@@ -79,7 +93,7 @@ partial dictionary CredentialCreationOptions {
 
 dictionary PaymentCredentialCreationOptions {
   required PublicKeyCredentialRpEntity rp;
-  required PaymentCredentialInstrument instrument;
+  required sequence<PaymentCredentialInstrument> instruments;
```

Comment:
Thanks @adrianhopebailie for putting up this pull request!

Since we don't yet have consensus whether the credential to instrument cardinality should be 1:1 or 1:N, and the pilot assumes 1:1, would it make sense to defer this generalization?